src/components/auth/LogoutButton.tsx

```diff
@@ -0,0 +1,22 @@
+"use client"
+
+import { useCallback } from "react"
+
+/**
+ * Logout button which navigates the browser to the logout API.
+ * The API will redirect to the SSO logout page and then back to the
+ * provided post-logout redirect (we attach `?post_logout=true`).
+ */
+export const LogoutButton = () => {
```

Comment:
use deafult export as this is a component and so it should use default exports according to our agreed instructions